src/CsvTextEditor/ViewModels/MainViewModel.cs

```diff
@@ -35,34 +39,23 @@ public MainViewModel(IProjectManager projectManager)
         public Project Project { get; set; }
 
         #region Methods
-        protected override Task InitializeAsync()
+        protected override async Task InitializeAsync()
         {
-            _projectManager.ProjectActivatedAsync += OnProjectActivated;
+            await base.InitializeAsync();
 
-            return base.InitializeAsync();
+            _projectManager.ProjectActivationAsync += OnProjectActivationAsync;
         }
```

Comment:
if you really need to initialize base class before subscribing, then everything done correctly. I just don't understand why you need it